common/file_watcher.cpp

```diff
@@ -46,7 +46,11 @@
  */
 // DEFINE_int32(recheck_removed_file_interval_ms, 2 * 1000,
 //              "The interval for checking removed files");
-const int kRecheckRemovedFileIntervalMs = 2000;
+
+const int MAX_TIME_INTERVAL_SLOTS = 10;
+const int* TIME_INTERVALS = new int[MAX_TIME_INTERVAL_SLOTS]{
+  5, 10, 20, 40, 80, 160, 320, 640, 1280, 2560
+};
```

Comment:
do you need deallocate these?

Reply:
Not until the process goes down.. so not absolutely necessary.